src/aux-server/aux-web/aux-player/scene/PlayerGame.ts

```diff
@@ -438,6 +447,13 @@ export class PlayerGame extends Game {
         );
     }
 
+    getMiniPortalHDRAddress() {
+        return this._getSimulationValue(
+            this.playerSimulations,
```

Comment:
@TroyceGowdy Shouldn't this be `miniSimulations` instead of `playerSimulations`?